scripts/json2wal/main.go

```diff
@@ -0,0 +1,75 @@
+/*
+	json2wal converts JSON file to binary WAL file.
+
+	Usage:
+			json2wal <path-to-JSON>  <path-to-wal>
+*/
+
+package main
+
+import (
+	"github.com/tendermint/go-amino"
+	"github.com/tendermint/tendermint/types"
+	cs "github.com/tendermint/tendermint/consensus"
+	"fmt"
+	"os"
+	"io"
+	"bufio"
+	"strings"
+)
+
+var cdc = amino.NewCodec()
+
+func init() {
+	cs.RegisterConsensusMessages(cdc)
+	cs.RegisterWALMessages(cdc)
+	types.RegisterBlockAmino(cdc)
+}
+
+
+func main() {
+	if len(os.Args) < 3 {
+		fmt.Println("missing arguments: Usage:json2wal <path-to-JSON>  <path-to-wal>")
+		os.Exit(1)
+	}
+
+	f, err := os.Open(os.Args[1])
+	if err != nil {
+		panic(fmt.Errorf("failed to open WAL file: %v", err))
+	}
+	defer f.Close()
+
+	walFile, err := os.OpenFile(os.Args[2],os.O_APPEND|os.O_WRONLY|os.O_CREATE,0666)
```

Comment:
don't think we should append. better crash if file already exists - `O_EXCL`

Reply:
ok.